python/mxnet/module/base_module.py

```diff
@@ -80,14 +80,14 @@ class BaseModule(object):
     """The base class of a module.
 
     A module represents a computation component. Modules are designed so that
-    they can be thoughtof as a computation "machine". Each model can run forward,
+    they can be thought of as a computation "machine". Each model can run forward,
     backward, update its parameters, etc. We aim to make the APIs easy to use,
     especially in the case when we need to use the imperative API to work with
     multiple modules (e.g. stochastic depth network).
 
     A module has several states:
 
-    - Initial state: Memory is not allocated yet, thus the moule is not ready for computation yet.
+    - Initial state: Memory is not allocated yet, so the module is not ready for computation yet.
     - Binded: Shapes for inputs, outputs, and parameters are all known, memory has been allocated,
       and the module is ready for computation.
     - Parameters initialized: For modules with parameters, doing computation before initializing
```

Comment:
Since you are describing what this state of module means, you should first say that the "parameters are initialized."
Same comment applies to the Optimizer installed state below.